Common/DataModel/EventSelection.h

```diff
@@ -123,20 +93,21 @@ DECLARE_SOA_COLUMN(NTracklets, nTracklets, int);        //! Tracklet multiplicit
 DECLARE_SOA_COLUMN(Sel7, sel7, bool);                   //! Event selection decision based on V0A & V0C
 DECLARE_SOA_COLUMN(Sel8, sel8, bool);                   //! Event selection decision based on TVX
 DECLARE_SOA_COLUMN(FoundFT0, foundFT0, int64_t);        //! FT0 entry index in FT0s table (-1 if doesn't exist)
+DECLARE_SOA_COLUMN(FoundFV0, foundFV0, int64_t);        //! FV0 entry index in FV0A table (-1 if doesn't exist)
```

Comment:
Is this a index to the other table as it says?
It should then be int only.
And why not declare it as index column?

Reply:
Indeed,I changed both foundFT0 and foundFV0 types to int32_t.
I am not sure whether is should be defined as index column. I will discuss with Anton. I suggest to merge it as it is now so that people could start using evsel with pilot data.